app/views/inventory_reconciliations/_incomplete_deltas.html.erb

```diff
@@ -19,14 +19,14 @@
 
       <tbody>
         <% @reconciliation.deltas.each do |delta| %>
-          <tr class="<%= delta.row_css_class %>" <%= delta.warning_path %> <%= delta.warning_text %>>
+          <%= reconciliation_delta_table_row(delta) %>
```

Comment:
See above requested change and modify to a `do...end` and replace ending `</tr>` with `<% end %>`